totaladapters/src/main/java/pl/arturborowy/adapters/common/contracts/OnItemListenerHolder.kt

```diff
@@ -0,0 +1,12 @@
+package pl.arturborowy.adapters.common.contracts
+
+import pl.arturborowy.adapters.common.interfaces.ClickableViewAdapter
+
+internal interface OnItemListenerHolder<ItemT> : ClickableViewAdapter<ItemT> {
```

Comment:
I'm not sure if you should break the naming convention of "Adapter" inheritance tree. I mean I wouldn't expect some `ListenerHolder` to have anything in common with some adapters. Maybe this class shouldn't extend ClickableViewAdapter at all?

Reply:
tasked